auth/src/main/java/io/grpc/auth/GoogleAuthLibraryCallCredentials.java

```diff
@@ -56,6 +58,8 @@
  * Wraps {@link Credentials} as a {@link CallCredentials}.
  */
 final class GoogleAuthLibraryCallCredentials implements CallCredentials {
+  private static final String SERVICE_ACCOUNT_CLASS_NAME
+      = "com.google.auth.oauth2.ServiceAccountCredentials";
```

Comment:
Ok then, reflectively load the class and find the methods.  But don't put something in the imports that may not be available at class load time.